extensions/wikia/VideoHandlers/VideoFileUploader.class.php

```diff
@@ -167,7 +167,7 @@ public function getNormalizedMetadata() {
 		$metadata = json_encode( $metadata );
 		$metadata = \UtfNormal::toNFC( $metadata );
 
-		return json_decode( $metadata );
+		return json_decode( $metadata, true );
```

Comment:
`json_decode` returns a stdClass object by default, or an associate array if you pass the optional argument `true`. Having this return an object was causing the following fatal during ingestion:

`Fatal error: Cannot use object of type stdClass as array in /usr/wikia/source/app/extensions/wikia/VideoHandlers/videoInfo/VideoInfoHelper.class.php on line 54`